static/app/components/charts/miniBarChart.tsx

```diff
@@ -265,22 +272,12 @@ function MiniBarChart({
         updated.stack = 'stack1';
       }
       set(updated, 'itemStyle.color', colorList[i]);
-      set(updated, 'itemStyle.opacity', barOpacity); // Opacity of each bar
       set(updated, 'itemStyle.borderRadius', [1, 1, 0, 0]); // Rounded corners on top of the bar
-      set(updated, 'emphasis.itemStyle.opacity', 1.0);
```

Comment:
These settings were for the entire graph. We now set them manually per bar because we want to conditionally make the bar invisible (set opacity=0 and emphasis opacity=0) if the value of the bar is 0. This logic is reflected in `barChart.tsx`